roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -65,6 +65,6 @@
 
 - name: Set docker version to be installed
   set_fact:
-    docker_version: "{{ '1.9.1' }}"
+    docker_version: "{{ '1.10.3' }}"
```

Comment:
to be determined, this is mostly just to get QE a version that installs with docker-1.10 so we can come to a conclusion on whether or not we're going to ship docker-1.0